src/Kdyby/Monolog/DI/MonologExtension.php

```diff
@@ -46,7 +46,12 @@ public function loadConfiguration()
 	{
 		$builder = $this->getContainerBuilder();
 		$config = $this->getConfig($this->defaults);
-		$builder->parameters[$this->name] = array('name' => $config['name']);
+
+		if (array_key_exists($this->name, $builder->parameters)) {
+			$builder->parameters[$this->name]['name'] = $config['name'];
```

Comment:
to be honest, there is really no point in this anymore... the intention was to somehow share the name with SyslogHandler so it doesn't have to be duplicated, but I haven't quite figured out yet, how to pass it, since this is a chicken and egg problem...